src/Uno.Templates/content/unoapp/MyExtensionsApp._1/Presentation/MainPage.xaml

```diff
@@ -24,6 +24,12 @@
 					HorizontalAlignment="Center"
 					VerticalAlignment="Center"
 					Spacing="16">
+<!--#if (mauiEmbedding)-->
+		<embed:MauiHost x:Name="MauiHostElement"
```

Comment:
@dansiegel as per previous comment - do you want to move namespaces to top of file?